src/Microsoft.Health.Dicom.Core/Messages/CustomTag/AddCustomTagResponse.cs

```diff
@@ -0,0 +1,29 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Health.Dicom.Core.Features.CustomTag;
+
+namespace Microsoft.Health.Dicom.Core.Messages.CustomTag
+{
+    public class AddCustomTagResponse
+    {
+        private readonly IEnumerable<CustomTagEntry> _customTags;
+        private readonly string _job;
+
+        public AddCustomTagResponse(IEnumerable<CustomTagEntry> customTags, string job)
+        {
+            _customTags = customTags;
```

Comment:
Should we ensure these aren't null/empty?

Reply:
not sure about that. Checked several XXResponse (e.g: https://github.com/microsoft/dicom-server/blob/master/src/Microsoft.Health.Dicom.Core/Messages/Store/StoreResponse.cs), none of them validate the parameters.

Reply:
My best guess is that these type of classes are data model, so any input should be acceptable. The validation could be in real business logic.